syntax="proto3";

package example;

message Message{
    string user_name=1;
    string user_message=2;
    int32 letter_count=3;
}